### DISCUSSION TITLE
涉及反射的类需要一个个写吗？

### DISCUSSION THREAD

**Discussion by xiayishusheng** [westinyang/java-graalvm-start#5] [Q&A] [answered]
涉及反射的类需要一个个写吗？

**Comment by westinyang** [comment, marked as answer]
可以通过native-image提供的agent模式运行程序，跑一边自动监听生成所需要的反射、资源、等配置文件再打包。除此之外，其实gluon提供的javafx打包插件、SpringNative插件都已经做好了大部分的反射、资源、动态代理的自动生成。
